doojon.js/src/breadboard.js

```diff
@@ -0,0 +1,147 @@
+export default class Container {
```

Comment:
This idea with containers and servers looks so great! Thanks to Perl's package BreadBoard and to his maintainer!